Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.Shared.cs

```diff
@@ -242,46 +192,31 @@ public void DrawTooltip_WithFollowingBoxAnnotation ()
       Helper.RunScreenshotTestExact<ElementScope, ScreenshotTest>(element, ScreenshotTestingType.Both, test);
     }
 
+    [Test]
+    public void TooltipBounds_NotDrawnYet_ThrowsInvalidOperationException ()
+    {
+      var tooltipAnnotation = new ScreenshotTooltipAnnotation("title", ScreenshotTooltipStyle.Chrome, WebPadding.None);
+
+      Assert.That(
+          () => tooltipAnnotation.TooltipBounds,
+          Throws.InstanceOf<InvalidOperationException>().With.Message.EqualTo("The tooltip needs to be drawn before its bounds can be accessed."));
+    }
     /// <summary>
-    /// Tests that the <see cref="ScreenshotTooltipAnnotation"/> is annotated correctly.
+    /// Tests that the <see cref="ScreenshotBadgeAnnotation"/> is drawn correctly.
```

Comment:
Since there's a missing blank line anway, please move to original position. makes blaming / DIFF-check eaiser.